cmake/HPX_AddTest.cmake

```diff
@@ -179,7 +173,10 @@ function(add_hpx_test category name)
         add_test(NAME "${_full_name}" COMMAND ${cmd} "-p" "lci" "-r" "mpi"
                                               ${args}
         )
-        set_tests_properties("${_full_name}" PROPERTIES RUN_SERIAL TRUE)
+        set_tests_properties(
+          "${_full_name}" PROPERTIES RUN_SERIAL TRUE ENVIRONMENT
+                                     "GASNET_PSHM_NODES=2"
+        )
```

Comment:
Should this environment variable added only if running using the gasnet parcelport?